packages-next/keystone/src/lib/createImagesContext.ts

```diff
@@ -48,14 +49,17 @@ const getImageMetadataFromBuffer = async (buffer: Buffer) => {
   return { width, height, filesize, extension: ext };
 };
 
-const getBlurhashFromBuffer = async (
-  buffer: Buffer,
-  width: number,
-  height: number
-): Promise<string> => {
-  return '';
+const getBlurhashFromBuffer = async (buffer: Buffer): Promise<string> => {
+  const SIZE = 50;
```

Comment:
maybe move this out of the function scope?

Reply:
@rohan-deshpande  ideally we want to not have the size as a magic number at all. We added this because we weren't able to get automatic width and height calculations that were compliant with blurHash at the time. (it was throwing errors). 

Currently the blurHash won't respect the aspect ratio of the original image, is this something you could pick up Rohan? 
Otherwise I'm happy to take a look at it after the remaining UI work is done

Reply:
It shouldn't be but you may have to end up just picking something. I'll check it out.